.github/workflows/build-test-lint.yml

```diff
@@ -27,9 +27,21 @@ jobs:
                 pip install --upgrade pip
                 pip install black
                 sudo pip install flake8
+                sudo pip install pip-audit
                 pip install -r requirements.txt -r requirements/dev.txt
               working-directory: xero-python
 
+            - name: Run audit
+              run: |
+
+               echo "running audit on requirements.txt...."
+               pip-audit -r requirements.txt
```

Comment:
can we combine and try like this?
`pip-audit -r requirements.txt -r requirements/dev.txt`